playbooks/operationalizing/ldap_group_sync.adoc

```diff
@@ -0,0 +1,341 @@
+---
+---
+= LDAP Group Synchronization
+Red Hat Containers & PaaS CoP
+v1.0, 2017-06-10
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
```

Comment:
We have an [issue](https://github.com/redhat-cop/openshift-playbooks/issues/128) to address this. i'd like to see it done sooner rather than later